x-pack/metricbeat/module/oracle/_meta/Dockerfile

```diff
@@ -1,3 +1,6 @@
 FROM docker.elastic.co/observability-ci/database-enterprise:12.2.0.1
 
-HEALTHCHECK --interval=1s --retries=90 CMD /usr/bin/echo 'select 1' | /u01/app/oracle/product/12.2.0/dbhome_1/bin/sqlplus sys/Oradoc_db1@localhost:1521/ORCLPDB1.localdomain AS SYSDBA
+ENV ORACLE_BASE=/db/oracle
```

Comment:
This was important to avoid some errors like: (TNS: lost contact) https://www.experts-exchange.com/articles/33072/ORA-12547-TNS-lost-contact-sqlplus-as-sysdba-Oracle-12-2-0-1.html